src/en/proposals/fairlysadpanda-revolutionaries.md

```diff
@@ -0,0 +1,333 @@
+4. The ability enters a cooldown.
+
+### Centcomm Uplink Terminal
+
+You win by compromising this terminal.
+
+To trigger this:
+
+1. Get at least one of the Heads ID cards. This must be their special ID cards, not HOP-manufactured ones.
+
+2. Input the cards and click the button to start the unlinking of the uplink.
+
+3. Wait some number of seconds for the uplink to be disabled. This alerts the entire station over comms that the Revs are doing this: I hope you have fortifed the bridge!
+
+# Role Outline: Revolutionary
+
+## How It Should Feel
+
+Plucked from your normal life by sudden desires for mutiny, you need to spread the good word across the station and assist your friends in overthrowing Centcomm. But don't be too suspicious: Security have ways of making you talk!
+
+## Goals
+
+### Spread The Revolution
+
+Use Propaganda posters and literature to spread the Influence of the Revolution and get into the harder-to-reach areas of the ship.
+
+### Free Yourself From Tyranny
+
+Ambush and arrest your department head; steal their ID for the greater good, and requisition their tools for the Revolution.
+
+### Don’t Get Gibbed By Sec
+
+Avoid being rumbled by Sec; they have guns and equipment that makes rumbling and smashing your mutiny possible.
+
+Core Mechanics
+
+### Propaganda
+
+The printers, posters and books of the station can be influenced by Revolutionaries, who can also make them via crafting and using the station’s printers. Sec, the HoP and the Captain are trained to identify revolutionary literature (it’s not obvious to the rest of the station), but it’s also possible for traitors and nukies to make them to throw Sec off their own activities.
+
+Propaganda slowly ticks up an exposed character’s Influence meter and allows Head Revs to quickly convert a compromised department if not dealt with.
+
+### Drugs
+
+Mindbreaker Toxin can be manufactured and administered as a bulk Influence dose.
+
+# Role Outline: The Security Team
+
+## How It Should Feel
+
+The crew might be plotting against you. Is the missing Chief Engineer due to incompetence, a traitor, or a mutiny? Dispatch officers to identify suspicious individuals - people who are in places they shouldn’t be, and confused people acting out-of-character. Find the spiders at the centre of the web and crush them. Or, robust your way to victory by reclaiming the Bridge from the mutineers.
+
+## Goals
+
+### Investigate Insurrection
+
+Why are your posters being torn down? Where’s the HoP’s ID? Why does that Clown seem to show up everywhere?
+
+### Crush The Mutiny
+
+Once you’re confident it’s a Rev round - the HoS being missing or his ID being stolen is a bit of a clue - you need to find out who it is who’s been doing all the converting and stop them. That’s easier said than done - even de-converted revolutionaries might not know who the ringleaders are, but arresting and interviewing them after deconversion is a good place to start.
```

Comment:
The lack of ANYONE knowing who the HRs are certainly helps the many cases where people have simply leaked who they are to sec

Reply:
That's the intent. Sec can keep the mutiny down but needs to round up the ringleaders by doing their job, or reacting well when the Revs make a play.

Because the powers are via an implant, an empty implanter can be used to ID a head rev if they get caught.